lib/figures/figure.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Figure
+  def find_possible_moves
```

Comment:
Where is information stored about where each of the figures is positioned? Is it stored in the `Board` class, or in the figures? 

In your code it seems to be stored in both of these positions, am I right? Because if that's the case, then it's redundant. Each information should just be stored in a single place.